app/Dockerfile

```diff
@@ -32,10 +32,16 @@ RUN apk add --update \
     py-pip \
     python \
     python-dev \
+    ruby \
+    ruby-bundler \
+    ruby-dev \
```

Comment:
It is needed. If I don't include it the gem winrm installation fails:
```
uccessfully installed builder-3.2.2
Successfully installed gyoku-1.3.1
Successfully installed nori-2.6.0
Successfully installed multi_json-1.11.2
Successfully installed little-plugger-1.1.4
Successfully installed logging-2.0.0
Successfully installed uuidtools-2.1.5
Successfully installed rubyntlm-0.4.0
Successfully installed httpclient-2.7.0.1
Building native extensions.  This could take a while...
ERROR:  Error installing winrm:
        ERROR: Failed to build gem native extension.

    /usr/bin/ruby -r ./siteconf20151211-23-rlcuyi.rb extconf.rb
mkmf.rb can't find header files for ruby at /usr/lib/ruby/include/ruby.h

extconf failed, exit code 1

Gem files will remain installed in /usr/lib/ruby/gems/2.2.0/gems/ffi-1.9.10 for inspection.
Results logged to /usr/lib/ruby/gems/2.2.0/extensions/x86_64-linux/2.2.0/ffi-1.9.10/gem_make.out
```

http://packages.ubuntu.com/trusty/ruby-dev says that this package is for compiling extension modules, which I suppose winrm's dependencies are doing.

However, `ruby-bundler` is not needed (and based on the description in http://packages.ubuntu.com/trusty/ruby-bundler that makes sense that its not needed). I removed that one.